### PR TITLE
Chore: eslint-config-eslint require node >= 8

### PR DESCRIPTION
**Pull request by aladdin-add**
**What is the purpose of this pull request? (put an "X" next to item)**

[x] Other, please explain:

**What changes did you make? (Give an overview)**

**Is there anything you'd like reviewers to focus on?**
I presume there is no much need to target the EOL node versions. and it seems too wordy to add something like `"eslint-plugin-node": "^6.0.1 || ^7.0.1 || ^8.0.0 || ^9.0.0..."` :)

### CODE REVIEW

**Comment by not-an-aardvark** [comment]
I think that would cause the ESLint release to be `Breaking`. The release type for `eslint-config-eslint` is selected manually without looking at commit messages.

**Comment by mysticatea** [comment]
I got it. Thank you for your answer!
